conformance/validate.yml

```diff
@@ -6,11 +6,11 @@ tests:
       expected: true
 
     - description: "Valid Tweet: 140 characters"
-      text: "A lie gets halfway around the world before the truth has a chance to get its pants on.  Winston Churchill (1874-1965) http://bit.ly/dJpywL"
+      text: "A lie gets halfway around the world before the truth has a chance to get its pants on. Winston Churchill (1874-1965) https://bit.ly/dJpywL"
```

Comment:
@jnievelt I felt since we check http is length 23 elsewhere it doesn't need to be checked here also. I don't have a strong opinion either way.